src/queries/get/InitQueries.ts

```diff
@@ -320,20 +319,10 @@ export async function getLinksConfig(
     });
   if (!appContextLinkRetrieval) return false;
   else {
-    const linkPredicates = [
-      parsePrefix(
-        "d-sgov-pracovní-prostor-pojem",
-        "odkazuje-na-assetový-kontext"
```

Comment:
I removed this - this was here for compatibility reasons during development of the very first milestone.